<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemManagerDlg</class>
 <widget class="QMainWindow" name="SystemManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="processes">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionLogout"/>
   <addaction name="actionStartProcess"/>
   <addaction name="actionStopProcess"/>
   <addaction name="actionRefresh"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionStartProcess"/>
    <addaction name="actionStopProcess"/>
    <addaction name="actionChangeProcessOperationMode"/>
    <addaction name="actionShutdownSystemController"/>
    <addaction name="actionSetProcessRuntimeParameters"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSystemManagerHelp"/>
    <addaction name="actionAboutSystemManager"/>
    <addaction name="actionStyle"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuControl"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLogin">
   <property name="icon">
    <iconset>
     <normaloff>:/SystemManager/Resources/login.bmp</normaloff>:/SystemManager/Resources/login.bmp</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset>
     <normaloff>:/SystemManager/Resources/logout.bmp</normaloff>:/SystemManager/Resources/logout.bmp</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset>
     <normaloff>:/SystemManager/Resources/refresh.bmp</normaloff>:/SystemManager/Resources/refresh.bmp</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionStartProcess">
   <property name="icon">
    <iconset>
     <normaloff>:/SystemManager/Resources/start.bmp</normaloff>:/SystemManager/Resources/start.bmp</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStopProcess">
   <property name="icon">
    <iconset>
     <normaloff>:/SystemManager/Resources/stop.bmp</normaloff>:/SystemManager/Resources/stop.bmp</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionChangeProcessOperationMode">
   <property name="text">
    <string>Change Process Operation Mode</string>
   </property>
  </action>
  <action name="actionShutdownSystemController">
   <property name="text">
    <string>Shutdown System Controller</string>
   </property>
  </action>
  <action name="actionSetProcessRuntimeParameters">
   <property name="text">
    <string>Set Process Runtime Parameters</string>
   </property>
  </action>
  <action name="actionSystemManagerHelp">
   <property name="text">
    <string>System Manager Help</string>
   </property>
  </action>
  <action name="actionAboutSystemManager">
   <property name="text">
    <string>About System Manager</string>
   </property>
  </action>
  <action name="actionWindows">
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="actionWindowsXP">
   <property name="text">
    <string>Windows XP</string>
   </property>
  </action>
  <action name="actionWindowsVista">
   <property name="text">
    <string>Windows Vista</string>
   </property>
  </action>
  <action name="actionFusion">
   <property name="text">
    <string>Fusion</string>
   </property>
  </action>
  <action name="actionStyle">
   <property name="text">
    <string>Style</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SystemManager.qrc"/>
 </resources>
 <connections/>
</ui>
